docs/1.5/extensions/footnotes.md

```diff
@@ -94,6 +95,14 @@ This extension can be configured by providing a `footnote` array with several ne
 
 This `string` option defines which HTML class should be assigned to rendered footnote backreference elements.
 
+### `backref_symbol`
+
+This `string` option sets the symbol used as the contents of the footnote backreference link. It defaults to `\League\CommonMark\Extension\Footnote\Renderer\FootnoteBackrefRenderer::DEFAULT_SYMBOL = '↩'`.
+
+If you want to use a custom icon, set this to an empty string `''` and use CSS to override the elements' contents. See the [Heading Permalink Extension docs](/1.5/extensions/heading-permalinks#adding-icons) for an example of how to do this.
```

Comment:
Minor nit/personal preference: generally speaking documentation should give examples for each feature/implementation, not reference something else saying it's "similar". While it's true that most could probably extrapolate the inference here, this can be wildly confusing for new developers. Let's just c&p the same example 
 down below and reword it to work for footnotes explicitly.

Reply:
Also, these doc changes should be in the 2.0 directory.

Reply:
Sounds good! Copy+paste was what I was trying to avoid, but I agree it's clearer to just be explicit.